StripeConnect/StripeConnect/Source/EmbeddedComponentManager.swift

```diff
@@ -33,7 +33,33 @@ public class EmbeddedComponentManager {
         ComponentAnalyticsClient(client: AnalyticsClientV2.sharedConnect,
                                  commonFields: $0)
     }
+    
+    @_spi(DashboardOnly)
+    var baseURL: URL = StripeConnectConstants.connectJSBaseURL
```

Comment:
does `baseURL` need to be public so it can be set from Dashboard?

Reply:
baseURL has been added to the initializer so I treated it the same as publicKeyOverride